src/ServicePulse.Host/vue/src/components/failedmessages/MessageView.vue

```diff
@@ -32,29 +37,29 @@ async function loadFailedMessage() {
   try {
     const response = await useFetchFromServiceControl("errors/last/" + id.value);
     if (response.status === 404) {
-      failedMessage.value = { notFound: true };
+      failedMessage.value = { notFound: true } as FailedMessageViewWithExtendedUIProperties;
     } else if (response.status !== 200) {
-      failedMessage.value = { error: true };
+      failedMessage.value = { error: true } as FailedMessageViewWithExtendedUIProperties;
     }
     const data = await response.json();
     const message = data;
-    message.archived = message.status === "archived";
-    message.resolved = message.status === "resolved";
-    message.retried = message.status === "retryIssued";
-    message.error_retention_period = moment.duration(configuration.value.data_retention.error_retention_period).asHours();
-    message.isEditAndRetryEnabled = configuration.value.edit.enabled;
+    message.archived = message.status === FailedMessageStatus.Archived;
```

Comment:
this needs to be checked if it's working - does the SC call return 'archived' or 'Archived'?